AcadeMeData/models.py

```diff
@@ -1,7 +1,8 @@
-from django.db import models
+from django.db import models, transaction
 from django.conf import settings
```

Comment:
please remove it is main through PR#83
I hope you understand that you had to build this PR based on PR #83